src/nordjylland_news/base_dataset_class.py

```diff
@@ -77,11 +77,14 @@ def data_path(self) -> str:
             str:
                 Data path.
         """
-        dirs = self.cfg["dirs"]
-        data = dirs["data"]
-        raw = dirs["raw"]
-        to_root = "../" * 3  # Just Hydra things
-        data_path = f"{to_root}{data}/{raw}/{self.dataset_name}.jsonl"
+        if self.dataset_name == "test":
+            data_path = f"{self.dataset_name}.jsonl"
+        else:
+            dirs = self.cfg["dirs"]
+            data = dirs["data"]
+            raw = dirs["raw"]
+            to_root = "../" * 3  # Just Hydra things
+            data_path = f"{to_root}{data}/{raw}/{self.dataset_name}.jsonl"
```

Comment:
Jeg vil råde dig til at bruge `pathlib` i stedet for strings til paths, da strings ikke virker på Windows.

Du importerer `Path` fra `pathlib`, og her vil du fx. skrive `Path(self.cfg.dirs.data) / self.cfg.dirs.raw / f"{self.dataset_name}.jsonl"`.

Derudover tror jeg ikke, at du har behov for `to_root`, da vi altid kalder vores scripts fra root alligevel.

Reply:
Når jeg bruger hydra, ændres root path til følgende

<img width="507" alt="image" src="https://user-images.githubusercontent.com/71556498/222695889-c1a16622-64b0-4a8e-b6d5-be5d07689f08.png">

`to_root` er derfor bare mit hack til at omgå dette. Forestiller mig, at der må være bedre best practice, men denne praksis har jeg ikke fundet ud af endnu.

Path gør brug af pathlib fra [8a654f1](https://github.com/alexandrainst/NordjyllandNews/pull/3/commits/8a654f11d8d561281c605512a248cb1096b5dd0b)

Reply:
Hm, spøjst. Kan du tjekke om root path stadig ændres hvis du kører det her script?

```
python src/scripts/build_image_caption_dataset.py hydra.run.chdir=false
```

Altså lige smid et breakpoint ind eller noget i den stil, og så kør det.

Reply:
Jeg kan se, at du har ændret `hydra.run.dir` til `.` i config'en. Fiksede det problemet?